tests/integration/test_spanner_chat_message_history.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from langchain_core.messages.ai import AIMessage
+from langchain_core.messages.human import HumanMessage
+
+from langchain_google_spanner import SpannerChatMessageHistory
+
+instance_id = os.environ["INSTANCE_ID"]
+database_id = os.environ["DATABASE_ID"]
```

Comment:
Can you add these values to the integration.cloudbuild.yaml file similar to https://github.com/googleapis/langchain-google-cloud-sql-mysql-python/blob/main/integration.cloudbuild.yaml#L25?

Reply:
We should also test this against both dialects

Reply:
We have a google and postgres databases set up see env vars in https://github.com/googleapis/langchain-google-spanner-python/blob/main/integration.cloudbuild.yaml#L28-L29